Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -1231,13 +1230,14 @@ void MediaPlayerPrivateGStreamer::notifyPlayerOfTrack()
     }
 
     Vector<TrackID> validStreams;
-    StringPrintStream getPadProperty;
-    getPadProperty.printf("get-%s-pad", typeName);
+    StringBuilder getPadPropertyBuilder;
+    getPadPropertyBuilder.append("get-"_s, typeName, "-pad"_s);
+    auto getPadProperty = getPadPropertyBuilder.toString();
```

Comment:
Same here, likely `makeString()` would do:


```suggestion
    auto getPadProperty = makeString("get-"_s, typeName, "-pad"_s);
```